Miki.Discord/DiscordClient.cs

```diff
@@ -185,8 +185,7 @@ public virtual async Task<IEnumerable<IDiscordGuildChannel>> GetChannelsAsync(ul
         /// <inheritdoc/>
         public virtual async Task<IDiscordChannel> GetChannelAsync(ulong id, ulong? guildId = null)
         {
-            var channel = await cacheHandler.Channels.GetAsync(id, guildId);
-
+            DiscordChannelPacket channel = (guildId == null) ? (await cacheHandler.Channels.GetAsync(id)) : (await cacheHandler.Channels.GetAsync(id, guildId));
```

Comment:
parenthesis are unnecessary
please put `?` and `:` on a newline. The project has 105 characters per line max-width.